README.md

```diff
@@ -327,6 +327,10 @@ An [Airplane](https://github.com/TECHNOVE/Airplane) server, which is a fork of T
 
 > **NOTE** only Java 8 and 11 are supported
 
+Extra variables:
+- `FORCE_REDOWNLOAD=false` : set to true to force the located server jar to be re-downloaded
```

Comment:
While looking at this I noticed the code you used was using the old `-n` logic rather than `isTrue`

https://github.com/itzg/docker-minecraft-server/blob/52cbf1fb64dd3d6c40444253a4fece07488456f7/start-deployAirplane#L22

Can you go ahead and change that?

Reply:
Done.